build_tools/python_deploy/build_windows_ci.sh

```diff
@@ -14,6 +14,7 @@ cmake -GNinja -Bbuild \
   -DLLVM_EXTERNAL_PROJECTS="torch-mlir" \
   -DLLVM_EXTERNAL_TORCH_MLIR_SOURCE_DIR="$PWD" \
   -DPython3_EXECUTABLE="$(which python)" \
+  -DTORCH_MLIR_ENABLE_STABLEHLO=OFF \
```

Comment:
It would be nice to get https://github.com/llvm/torch-mlir/blob/main/.github/workflows/ci.yml running Windows (and macOS) builds, given sufficiently powerful runners. (We're struggling with that in my project right now too). If the `pull_request` trigger is out of reach (due to slow builds / low runner capacity), nightly builds with the `schedule` trigger would be better than nothing.

I see Windows builds in https://github.com/llvm/torch-mlir/blob/main/.github/workflows/buildRelease.yml but that is only run manually and it hasn't been run in 9 months.

Reply:
I'm planning to submit a PR in https://github.com/llvm/torch-mlir-release to have Windows build running once this change is merged. The linux build is run nightly and I am planning to run the Windows on the same cadence as well. Eventually we want to have wheels for macOS too in the same fashion.